### PR TITLE
[Gamepad article] Remove Switch references, add iframe embed instead

### PR DESCRIPTION
**Pull request by tomayac**
CC: @PaulKinlan.

Changes proposed in this pull request:

- Remove Switch references
- Add iframe embed instead

### CODE REVIEW

File: src/site/content/en/blog/gamepad/index.md [hunk 1]
```diff
@@ -275,38 +274,25 @@ or moving any of its axis.
...
+[on GitHub](https://github.com/tomayac/chrome-dino-gamepad).
+Check out the gamepad polling implementation
+[in `trex-runner.js`](https://github.com/tomayac/chrome-dino-gamepad/blob/885eb6134805345bf31eeb9971830adeb84747ab/trex-runner.js#L529-L571)
+and note how it is emulating key presses.
+
+<div style="height: 420px; width: 100%;">
+  <iframe
+    src="https://tomayac.github.io/chrome-dino-gamepad/"
+    title="Chrome dino gamepad"
+    allow="gamepad"
```
Comment (tomayac) [inline src/site/content/en/blog/gamepad/index.md:292]:
```suggestion
    allow="gamepad; fullscreen"
```

File: src/site/content/en/blog/gamepad/index.md [hunk 2]
```diff
@@ -275,38 +274,25 @@ or moving any of its axis.
...
+You can play
+[Chrome dino](https://tomayac.github.io/chrome-dino-gamepad/)
+with your gamepad on this very site.
+The source code is available
+[on GitHub](https://github.com/tomayac/chrome-dino-gamepad).
+Check out the gamepad polling implementation
+[in `trex-runner.js`](https://github.com/tomayac/chrome-dino-gamepad/blob/885eb6134805345bf31eeb9971830adeb84747ab/trex-runner.js#L529-L571)
+and note how it is emulating key presses.
+
+<div style="height: 420px; width: 100%;">
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/gamepad/index.md:288]: Nit: Use "Iframe" web.dev component

File: src/site/content/en/blog/gamepad/index.md [hunk 3]
```diff
@@ -275,38 +274,25 @@ or moving any of its axis.
...
-   so you can easily toggle between **Automatic** and **Manual**.
+## Bonus: play Chrome dino on web.dev
+
+You can play
+[Chrome dino](https://tomayac.github.io/chrome-dino-gamepad/)
+with your gamepad on this very site.
+The source code is available
+[on GitHub](https://github.com/tomayac/chrome-dino-gamepad).
+Check out the gamepad polling implementation
+[in `trex-runner.js`](https://github.com/tomayac/chrome-dino-gamepad/blob/885eb6134805345bf31eeb9971830adeb84747ab/trex-runner.js#L529-L571)
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/gamepad/index.md:285]: Maybe remove `in` from the link. It looks weird.

image

Comment (tomayac) [inline src/site/content/en/blog/gamepad/index.md:285]:
```suggestion
Check out the gamepad polling implementation in
[`trex-runner.js`](https://github.com/tomayac/chrome-dino-gamepad/blob/885eb6134805345bf31eeb9971830adeb84747ab/trex-runner.js#L529-L571)
```

File: src/site/content/en/blog/gamepad/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/gamepad/index.md:288]: I don't think there exists one: https://web.dev/handbook/web-dev-components/

Comment (beaufortfrancois) [reply src/site/content/en/blog/gamepad/index.md:288]: There's one https://github.com/GoogleChrome/web.dev/blob/master/src/site/_includes/components/IFrame.js used by glitch https://github.com/GoogleChrome/web.dev/blob/master/src/site/_includes/components/Glitch.js

Comment (tomayac) [reply src/site/content/en/blog/gamepad/index.md:288]: With the still required div wrapper for responsiveness it's honestly not adding much value. Maybe that's why it's not documented. It makes a lot of sense for Glitch embeds, but not bare iframes.

Comment (beaufortfrancois) [reply src/site/content/en/blog/gamepad/index.md:288]: With "{Iframe}", `loading="lazy"` is automatically added ;) Your choice!

File: src/site/content/en/blog/gamepad/index.md [hunk 4]
```diff
@@ -275,38 +274,25 @@ or moving any of its axis.
...
+Check out the gamepad polling implementation in
+[`trex-runner.js`](https://github.com/tomayac/chrome-dino-gamepad/blob/885eb6134805345bf31eeb9971830adeb84747ab/trex-runner.js#L529-L571)
+and note how it is emulating key presses.
+
+<div style="height: 420px; width: 100%;">
+  <iframe
+    src="https://tomayac.github.io/chrome-dino-gamepad/"
+    title="Chrome dino gamepad"
+    allow="gamepad; fullscreen"
+    style="height: 100%; width: 100%; border: 0;">
```
Comment (tomayac) [inline src/site/content/en/blog/gamepad/index.md:293]:
```suggestion
    style="height: 100%; width: 100%; border: 0;"
    loading="lazy">
```

File: src/site/content/en/blog/gamepad/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/gamepad/index.md:288]: Just done that manually in https://github.com/GoogleChrome/web.dev/pull/4181/commits/814ecbcfac704ca0c116e8364da5e6cb90f3a117.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**